packages/Webkul/Velocity/src/Resources/views/shop/home/slider.blade.php

```diff
@@ -15,7 +15,7 @@
                 autoplay="true"
                 slides-per-page="1"
                 navigation-enabled="hide"
-                locale-direction="direction"
+                locale-direction="{{ $direction }}"
```

Comment:
The direction prop is already passed in the component. It should be like this,

`:locale-direction="direction"`

Reply:
Yep, thanks. 
added.